src/nvim/testdir/test_registers.vim

```diff
@@ -43,6 +43,9 @@ func Test_yank_shows_register()
 endfunc
 
 func Test_display_registers()
+    " Disable clipboard
+    let g:clipboard = {}
```

Comment:
This is fine but put in separate commit.

Reply:
Done.

Reply:
Forgot to ask.
Does this affect the other tests in this file in case that this fails?

Reply:
Please see it.

https://github.com/neovim/neovim/pull/12784#issuecomment-683495832

It is neovim only problem.

Reply:
Without your changes, I can reproduce that error on my machine depending on the clipboard state.

What I'm asking is if it impacts the other tests in case `Test_display_registers()` because `g:clipboard` is not restored. Vim tests typically restore the previous state of the test runner within the test such that the test must pass to not break the next tests.
It's fine if you don't know.

Reply:
OK.  I have added the restore.